components/dashboard/src/components/org-icon/consistent-classname.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+// Represents the max value our base16 guid can be, 32 "f"s
+const GUID_MAX = "".padEnd(32, "f");
```

Comment:
**issue**: Not sure if this was random and it's possibly minor regarding the entropy of the color selection, but when creating orgs serially, the backgrounds are almost clustered every two orgs.

<img width="253" alt="Screenshot 2023-02-13 at 20 36 04" src="https://user-images.githubusercontent.com/120486/218545231-1efdd55e-1fb6-40b4-9288-8cf7decc7816.png">

Reply:
Seems more random after trying this again—I'll keep an eye on this and keep you in the loop if I notice any issues.